packages/core/src/families/bitcoin/serializer.ts

```diff
@@ -1,30 +1,35 @@
 import BigNumber from "bignumber.js";
 import type { BitcoinTransaction, RawBitcoinTransaction } from "./types";
+import { schemaRawBitcoinTransaction } from "./validation";
 
 export function serializeBitcoinTransaction({
   family,
   amount,
   recipient,
   feePerByte,
+  opReturnData,
 }: BitcoinTransaction): RawBitcoinTransaction {
-  return {
+  return schemaRawBitcoinTransaction.parse({
```

Comment:
I don't think you should parse here as it's not done like that in other families and we do the validation on the server when receiving a `rawTransaction` in the `params`

Reply:
Found it. It makes sense.

https://github.com/LedgerHQ/wallet-api/blob/main/packages/server/src/internalHandlers/transaction.ts#L19